src/commands/convert/hdf2asff.ts

```diff
@@ -68,22 +84,25 @@ export default class HDF2ASFF extends Command {
               console.log(result.FailedFindings)
             }
           } catch (error) {
-            console.error(`Failed to upload controls: ${error}`)
+            if (typeof error === 'object' && _.get(error, 'code', false) === 'NetworkingError') {
+              console.error(`Failed to upload controls: ${error}; Use --certificate to provide your own SSL intermediary certificate (in .crt format) or use the flag --insecure to ignore SSL`)
+            } else {
+              console.error(`Failed to upload controls: ${error}`)
+            }
           }
         }),
       ).then(async () => {
-        profileInfoFinding.UpdatedAt = new Date().toISOString()
-        const profileInfoUploadCommand = new BatchImportFindingsCommand({
-          Findings: [profileInfoFinding],
-        })
-        const result = await client.send(profileInfoUploadCommand)
-        console.info(`Statistics: ${profileInfoFinding.Description}`)
-        console.info(
-          `Uploaded Results Set Info Finding(s) - Success: ${result.SuccessCount}, Fail: ${result.FailedCount}`,
-        )
-        if (result.FailedFindings?.length) {
-          console.error(`Failed to upload ${result.FailedCount} Results Set Info Finding`)
-          console.log(result.FailedFindings)
+        if (profileInfoFinding) {
+          profileInfoFinding.UpdatedAt = new Date().toISOString()
+          const result = await client.batchImportFindings({Findings: [profileInfoFinding as any]}).promise()
```

Comment:
i mean it's not that much of an improvement, but instead of just `any` can't we do something like `Record<string, any>` so it's at least obvious it should be an object